plugins/modules/lxd_container.py

```diff
@@ -172,6 +173,16 @@
         required: false
         aliases: [ cert_file ]
         type: path
+    verify:
```

Comment:
The usual name for certificate validation toggles is `validate_certs`.
```suggestion
    validate_certs:
```